client/src/types.ts

```diff
@@ -86,6 +87,18 @@ export interface Lesson {
   deleted?: boolean;
 }
 
+export interface Media {
+  url: string;
+  type: string;
+  props?: Array<{ name: string; value: string }>;
+}
+
+export enum MediaType {
+  NONE = "none",
```

Comment:
do we need `NONE` type? that value should never be used and I would prefer the empty string if we did need it